src/components/MerchantsPage/ContributionBar.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import styled from 'styled-components';
+
+interface Props {
+  totalDonations: number;
+  totalVouchers: number;
```

Comment:
Even though they're called vouchers externally, can we call them gift cards internally because that's how they're modeled and I don't wanna change the name of our datatbase tables

Reply:
yea! i can change this back